Packages.props

```diff
@@ -8,7 +8,6 @@
     <PackageReference Update="Azure.Messaging.EventHubs" Version="5.6.2" />
```

Comment:
Not sure if we need this guy here, or should we remove it too?

Reply:
++, removed